### PR TITLE
Refresh page after search

### PR DESCRIPTION
**Pull request by BrookJeynes**
Page will now refresh after searching.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ssw.megamenu",
-  "version": "4.3.2",
+  "version": "4.4.2",
```
Comment (Harry-Ross) [inline package.json:3]:
```suggestion
  "version": "4.4.0",
```

File: lib/components/Search/Search.tsx [hunk 2]
```diff
@@ -31,9 +31,7 @@ export const Search: React.FC<SearchProps> = ({ url, callback }) => {
     }
   }, [isOpen]);
 
-  const performSearch = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
```
Comment (Harry-Ross) [inline lib/components/Search/Search.tsx:35]: Why was this removed?

Comment (BrookJeynes) [reply lib/components/Search/Search.tsx:35]: On Rules, we pass a callback through the megamenu to obtain the search result and redirect the user to the search page. If the user cmd+k searches, it will "redirect" to the same page with a different query param but not refresh the page results. 

The search originally refreshed the page but I removed that as part of the callback change, this is simply reverting it.

**Review by Harry-Ross — APPROVED** [review]
lgtm 🚢

**Review by JackDevAU — APPROVED** [review]

File: lib/components/Search/Search.tsx [hunk 3]
```diff
@@ -43,8 +41,6 @@ export const Search: React.FC<SearchProps> = ({ url, callback }) => {
         )}`;
         window.open(searchUrl, "_blank");
       }
-      setIsOpen(false);
-      setSearchTerm("");
```
[the remark this answers is not in the document]
Comment (BrookJeynes) [reply lib/components/Search/Search.tsx:47]: The page reloading sets them back to their defaults which is essentially all that code was doing (to compensate not refreshing).
